src/ConsoleArgument.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console;
+
+use Attribute;
+use Tempest\Support\ArrayHelper;
+
+#[Attribute(Attribute::TARGET_METHOD | Attribute::TARGET_PROPERTY | Attribute::TARGET_PARAMETER)]
+final class ConsoleArgument
+{
+    /** @var string[] */
+    private array $helpMessages;
+
+    /**
+     * @param string|string[] $help
+     * @param string[] $aliases
+     */
+    public function __construct(
+        array|string $help = [],
```

Comment:
I'm still not convinced that an array has value here. Can you can an example of the complex console applications that you mention where an array of string lines would be better than a multiline string?

Reply:
For arguments, this would probably just be a convenience - I think it's nicer experience in user-land if I can just 
 do 

#[ConsoleArgument(help: ['a', 'b', 'c'])] rather than #[ConsoleArgument(help: 'a' . PHP_EOL . 'b' . PHP_EOL . 'c')]

For commands - I think having multi line help is useful - I think this could encourage people to create more descriptive commands that are easier to understand from command line - i.e

![image](https://github.com/tempestphp/tempest-console/assets/43403182/96ed23ed-6aff-40c2-a473-95b83e5a28e3)

Reply:
You'd never do this though?

```
#[ConsoleArgument(help: 'a' . PHP_EOL . 'b' . PHP_EOL . 'c')]
```

You'd write it like so:

```php
#[ConsoleArgument(help: <<<TXT
This is a multiline help message
This is the second line
This is the third
TXT)]
```

In comparison:

```php
#[ConsoleArgument(help: [
'This is a multiline help message',
'This is the second line',
'This is the third',
]]
```